client/components/ThemeMenu.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import FormControl from '@mui/material/FormControl';
+import Select, { SelectChangeEvent } from '@mui/material/Select';
+
+import { useGlobalContext } from "../context/GlobalContext";
+
+export default function ThemeMenu() {
```

Comment:
We should add a test for this component once it is finalized and before we merge this PR.